templates/cli/lib/commands/init.js.twig

```diff
@@ -71,20 +71,54 @@ const initFunction = async () => {
     parseOutput: false
   })
 
-  let command = `
-    mkdir -m 777 -p 'functions/${answers.name}' && \
-    cd 'functions/${answers.name}' && \
+    let initFirstCommand = `
+    cd '${answers.name}' && \
     git init && \
     git remote add -f origin https://github.com/{{ sdk.gitUserName }}/functions-starter && \
-    git config core.sparseCheckout true && \
-    echo '${answers.runtime.id}' >> .git/info/sparse-checkout && \ 
-    git pull origin main && \
-    rm -rf .git && \
-    mv ${answers.runtime.id}/* . && \
-    rm -rf ${answers.runtime.id}`;
+    git config core.sparseCheckout true`;
+
+  let initSecondCommand = `
+    cd '${answers.name}' && \
+    git pull origin dev && \
+    git checkout dev`;
+
+  // Convert commands from unix based ones to windows based ones
+  if (process.platform == 'win32') {
+    initFirstCommand = initFirstCommand.replace(/'/g, '"').replace(/\\/g, '').replace(/\n/g, '').replace(/  +/g, ' ');
+    initFirstCommand = 'cmd /c "' + initFirstCommand + '"';
+    initSecondCommand = initSecondCommand.replace(/'/g, '"').replace(/\\/g, '').replace(/\n/g, '').replace(/  +/g, ' ');
+    initSecondCommand = 'cmd /c "' + initSecondCommand + '"';
+  }
```

Comment:
Why do we need platform specific logic here ?

Reply:
Windows commands work differently from UNIX commands and don't support things such as new lines and `&&` is only supported in CMD not Powershell